src/edge/templates/edge/edge.html

```diff
@@ -28,7 +28,7 @@
     <script>
       window["_fs_debug"] = false;
       window["_fs_host"] = "{{ FULLSTORY_URL }}";
-      window["_fs_script"] = "fullstory.com/s/fs.js";
+      window["_fs_script"] = "edge.fullstory.com/s/fs.js";
```

Comment:
I suppose this is just a coincidence that fullstory also happens to call this edge and it has nothing to do with this repo called edge, right?

Reply:
Yup, exactly. Their "edge" is in the sense of "Edge CDN".